test/bigtest/network/scenarios/fetch-match-profiles-success.js

```diff
@@ -0,0 +1,232 @@
+
+export default server => {
+  server.create('match-profile', {
+    id: 'cdf0ca3a-b515-4abd-82b6-48ce65374963',
+    name: 'POL-MARC',
+    description: 'Use for POL in 990 $p',
+    tags: {

```

Comment:
Tags are not needed to be overridden for every item, so specify them for one or two but not for all 8 items.

Reply:
Don't agree.
Names are unique